src/assets/_project/_blocks/components/forms/recaptcha.js

```diff
@@ -202,17 +212,11 @@ import keys from '../../data/qg-google-keys';
         onloadRecaptcha,
         'Recaptcha unavailable',
       );
+      $('.qg-feedback-toggle').one('click', ajaxMethod);
```

Comment:
how does this fix it?

Reply:
In the case of 'contact us' and 'footer feedback' it is hitting a condition where footer ajax call is not included.

We need to simplify this script at some point, there are a lot of conditions. Initially, I thought 'loadfooter' should always be evaluated whenever we load the footer but this is not evaluated on some conditions.